components/README.md

```diff
@@ -33,3 +33,155 @@ Note the naming convention for client code and runtime code&mdash;for a task nam
 
 See how to [use the Kubeflow Pipelines SDK](https://www.kubeflow.org/docs/pipelines/sdk/sdk-overview/)
 and [build your own components](https://www.kubeflow.org/docs/pipelines/sdk/build-component/).
+
+## Index of components
+ / [_converters](https://github.com/kubeflow/pipelines/tree/b63dd3f53237f74ae7f965441fd97698fafba7c9/components/_converters) / [ApacheParquet](https://github.com/kubeflow/pipelines/tree/b63dd3f53237f74ae7f965441fd97698fafba7c9/components/_converters/ApacheParquet) / [from_ApacheArrowFeather](https://github.com/kubeflow/pipelines/tree/b63dd3f53237f74ae7f965441fd97698fafba7c9/components/_converters/ApacheParquet/from_ApacheArrowFeather) / [Convert apache arrow feather to apache parquet](https://raw.githubusercontent.com/kubeflow/pipelines/b63dd3f53237f74ae7f965441fd97698fafba7c9/components/_converters/ApacheParquet/from_ApacheArrowFeather/component.yaml)
```

Comment:
nit: We might need a new line after each entry. Currently it looks very busy. WDYT?

Reply:
Thank you for catching this. The markdown preview in VSCode is incorrect =(
Fixed.